### ISSUE TITLE
Channel change stutter.

### ISSUE THREAD

**Issue by starvald** [khaytsus/gqrx-scan#19]
I'm scanning between some channels but on the same centre frequency but gqrx acts like it's re-tuning so i see a stutter and a hear a small squelch break.
it does this on airspy too but it's able to retune more quickly, i now also have an sdrplay rsp2 and the stutter is more apparent.

**Comment by khaytsus** [comment]
This unfortunately I have zero control over.  I've had this happen too and I believe it's related to rtl-sdr libraries.  When I've seen it happen is when I have been messing with direct sampling and such.

Are you also seeing the tuning, even within the same bandpass (ie: hardware freq isn't changing, just the demodulation frequency) go much slower than usual?  This is the usual symptom that I see, plus it having weird other issues.

**Comment by starvald** [comment]
yes the hardware frequency stays the same just the demodulation frequency changes and it is slower.
wouldn't be too bad apart from the sound, squelch sounding on every change means it's too annoying to have the volume up. 
this is the output from gqrx run from bash.
https://paste.ubuntu.com/24433551/

i've also tried Marzona's rig-remote and that doesn't stutter and is faster changing frequency, so i though it could be a problem with gqrx-scan.

**Comment by khaytsus** [comment]
Interesting..  I wonder what could be different.  I even tried using the TCP interface directly (telnet) and got the slow, laggy response when I was messing with rtlsdr's libs.  Only after I'd gone back to the regular ones did GQRX start tuning fast without delays and such.  

I assume it does the same on channel scans and frequency scans right?

**Comment by starvald** [comment]
did a scan between 166MHz and 453MHz
on gqrx-scan it stutters when changing hardware frequency.
with rig-remote i get the changes as the above pastebin but it doesn't stutter and is still as fast as where only the demod freq changes.
